xapian-core/api/omenquireinternal.h

```diff
@@ -313,6 +313,21 @@ class MSet::Internal : public Xapian::Internal::intrusive_base {
 	/** Fetch items specified into the document cache.
 	 */
 	void fetch_items(Xapian::doccount first, Xapian::doccount last) const;
+
+	/** Updates the weight corresponding to the document indexed at position Xapian::docount i with double wt_.
+	 *  Updates the max_possible and max_attained for the MSet.
+	 *  set_item_weight must be called upon to update the weight of every document in MSet
+	 *  in the same order as that of the documents present in MSet to avoid miscalculation of
+	 *  max_attained and max_possible.
+	 *
+	 *  @param i Xapian::docount i, position of the document whose weight is being updated.
+	 *  @param wt_ double wt_, new weight assigned to the document.
```

Comment:
This isn't quite how `@param` works; check out the [relevant doxygen documentation](https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdparam) for details.